planet/src/main/webapp/places.js

```diff
@@ -113,8 +118,15 @@ function addPlaceDetails(places) {
           console.log("Returned place contains no geometry");
           return;
         }
-        // Add search results to a list
-        placeInfo.push(place.name + "\n" + "Rating: " + place.rating + "\n"); 
+
+        // Check if rating is defined
+        if (place.rating != undefined) {
+             // Add search results to a list
+            placeInfo.push(place.name + "\n" + "Rating: " + place.rating + "\n");  
+        }
+        else {
+            placeInfo.push(place.name + "\n");  
+        }
```

Comment:
This is an example where you can make your PR's smaller and easier to land. The PR is titled "Add photos to place result cards" so the rating here isn't relevant.
I'm gonna let this one slide since it's short and straightforward.

Reply:
Okay oops! This code was already present but I think since I moved its placement it counted as a change